insights/specs.py

```diff
@@ -149,6 +149,7 @@ def docker_container_ids(broker):
 foreman_proxy_conf = sf.simple_file("/etc/foreman-proxy/settings.yml", name="foreman_proxy_conf")
 foreman_proxy_log = sf.simple_file("/var/log/foreman-proxy/proxy.log", name="foreman_proxy_log", alias="foreman_proxy.log")
 foreman_satellite_log = sf.simple_file("/var/log/foreman-installer/satellite.log", name="foreman_satellite_log", alias="foreman_satellite.log")
+foreman-ssl_access_ssl.log = sf.first_of([sf.simple_command("var/log/httpd/foreman-ssl_access_ssl.log"),sf.simple_file(r"sos_commands/foreman/foreman-debug/var/log/httpd/foreman-ssl_access_ssl.log", context=HostArchiveContext)], name="foreman-ssl_access_ssl.log")
```

Comment:
In this insights/specs.py
1. you should rename it to `foreman_ssl_access_ssl_log`, since `-` and `.` are not permitted in a variable name.
E.g.
```
foreman_ssl_access_ssl_log = sf.first_of(..., name="foreman_ssl_access_ssl_log", alias="foreman-ssl_access_ssl.log")
```

2. the first ` sf.simple_command` should be `sf.simple_file` too.

Reply:
@xiangce : Done...Please review it.